### ISSUE TITLE
[Feature Request] Android: Bridge support

### ISSUE THREAD

**Issue by Pizzacus** [mullvad/mullvadvpn-app#1210]
Couldn't find any other issue about this feature. Is bridge/shadowsocks support planned on Android?

Alternatively, do you have an idea on how to make the app work with the Shadowsocks Android app previously used back when we didn't have a Mullvad app?

**Comment by kah0922** [comment]
Before either of those things are supported, OpenVPN/TCP support needs to be supported in the Mullvad Android App.

**Comment by Pizzacus** [comment]
Wireguard doesn't work through bridges? Hmm, that makes sense then.

**Comment by kah0922** [comment]
Any updates on this?

**Comment by faern** [comment]
We want to support obfuscation (via for example shadowsocks) on mobile eventually. But this is currently blocked on a few things. Most notably that WireGuard is UDP only and most obfuscation protocols can't really handle that.

We are not going to go backwards in time and add OpenVPN support to mobile. Instead we will find a way to obfuscate WireGuard. But there is no exact timeline for this.

**Comment by faern** [comment]
I'm closing this as duplicate of #1049
